netsim/augment/__init__.py

```diff
@@ -1,13 +1,13 @@
 # Import the whole augmentation tree
 
+from . import devices
 from . import nodes
 from . import groups
 from . import links
-from . import devices
 from . import plugin
 from . import topology
-from . import main
 from . import config
 from . import components
 from . import tools
 from . import validate
+from . import main
```

Comment:
Talking about reordering chairs on a deck... :)

Reply:
I had a circular dependency issue and thought ordering things in approximate order of dependencies might solve that. Of course it didn't ;)